pkgs/applications/misc/mako/default.nix

```diff
@@ -22,6 +22,15 @@ stdenv.mkDerivation rec {
     "-Dsd-bus-provider=libsystemd"
   ];
 
+  preFixup =
+    let
+      # make sure busctl and jq are available
+      runtimePath = lib.makeBinPath [ systemd jq ];
+    in
+    ''
+      sed -i "2 i export PATH=${runtimePath}:\$PATH" $out/bin/makoctl
```

Comment:
Any reason not to use a wrapper?

I guess this isn't that brittle since we're just adding a line at the top

Maybe also add `\n` at least to the end?

> When we use sed‘s “a” or “i” commands, it’s a good practice to precede the new content with a backslash.

https://www.baeldung.com/linux/insert-line-with-spaces-sed

Reply:
The script is already being wrapped by `wrapGAppsHook`, so I didn't want to wrap it another time.

I opted to use `gappsWrapperArgs` instead, which does have the side effect of also doing this for the main mako binary. But that shouldn't really cause issues here.